### PR TITLE
changing the HLIP address used in apipa endpoint

### PR DESCRIPTION
**Pull request by vinayakawanti**
Using the different HLIP prefix.

### CODE REVIEW

File: network/endpoint_windows.go
Comment (tamilmani1989) [inline network/endpoint_windows.go:46]: why this change is required?
 
@ashvindeodhar will this affect windows multitenancy hostnc connectivity?

Comment (vinayakawanti) [reply network/endpoint_windows.go:46]: HLIP prefix 169.254.255.192/26 is allocated for dedicated honeycomb use.
The previous apipaHostLevelIP was in different subnet. 

Just FYI, this change is just for honeycomb-dev branch will be never merged to master.

Comment (tamilmani1989) [reply network/endpoint_windows.go:46]: ok then its fine.. lgtm.. lets wait for @aegal approval too

Comment (ashvindeodhar) [reply network/endpoint_windows.go:46]: is the plan to maintain different fork for honeycomb? @vinayakawanti / @aegal

Comment (vinayakawanti) [reply network/endpoint_windows.go:46]: nope, this branch won't be used after phase-0 of honeycomb project.

**Review by aegal — APPROVED** [review]
lgtm

**Comment by tamilmani1989** [comment]
@vinayakawanti looks like this PR not merged yet. If you don't want this PR to be merged, please close the PR

**Pull request merged**
